### PR TITLE
[lidarr] refactor of exportarr sidecar and podmonitor

### PR DESCRIPTION
**Pull request by jr0dd**
If these changes look good I'll update sonarr and radarr as well.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by onedr0p** [comment]
I think you can keep it consistent and make the key name `exporter` instead of `exportarr`

**Comment by jr0dd** [comment]
> I think you can keep it consistent and make the key name `exporter` instead of `exportarr`, we can keep it like the qb one.

Yeah no problem. Are there any other additional env vars for exportarr you think should be included?

File: charts/stable/lidarr/values.yaml [hunk 1]
```diff
@@ -66,35 +66,25 @@ persistence:
...
+    repository: ghcr.io/onedr0p/exportarr
+    # -- image tag
+    tag: v0.6.1
+    # -- image pull policy
+    pullPolicy: IfNotPresent
+  env:
+    # -- metrics port
+    port: 32123
+    # -- Set to true to enable gathering of additional metrics (slow)
+    addMetrics: false
```
Comment (onedr0p) [inline charts/stable/lidarr/values.yaml:88]:
```suggestion
    additionalMetrics: false
```

File: charts/stable/lidarr/values.yaml [hunk 2]
```diff
@@ -66,35 +66,25 @@ persistence:
...
+    # -- Set to true to enable gathering unknown queue items
+    unknownItems: false
```
Comment (onedr0p) [inline charts/stable/lidarr/values.yaml:90]:
```suggestion
    unknownQueueItems: false
```

**Comment by onedr0p** [comment]
I think you got all the ones that should be configurable. Just made a comment on to rename the 2 default ones

File: charts/stable/lidarr/templates/common.yaml [hunk 3]
```diff
@@ -1 +1,36 @@
...
+        value: "{{ .Values.exporter.env.additionalMetrics }}"
+      - name: ENABLE_UNKNOWN_QUEUE_ITEMS
+        value: "{{ .Values.exporter.env.unknownQueueItems }}"
+    ports:
+      - name: metrics
+        containerPort: {{ .Values.exporter.env.port }}
+    volumeMounts:
+      - name: config
+        mountPath: /config
+        readOnly: true
```
Comment (bjw-s) [inline charts/stable/lidarr/templates/common.yaml:30]: This can potentially break when there is the `config` persistence item is disabled.

Comment (jr0dd) [reply charts/stable/lidarr/templates/common.yaml:31]: ok should be fixed

**Review by bjw-s — APPROVED** [review]

**Comment by jr0dd** [comment]
@bjw-s or @onedr0p  So for whatever reason this didn't land in the repo. Sonarr and radarr went through just fine.

**Comment by onedr0p** [comment]
I see in lidarr-12.0.0 in the github releases.

**Comment by onedr0p** [comment]
If it's not coming thru we probably need to bump it to `12.0.1`.

**Pull request merged**